sirepo/job_driver/__init__.py

```diff
@@ -200,12 +196,12 @@ def make_lib_dir_symlink(self, op):
 
     def pkdebug_str(self):
         return pkdformat(
-            '{}(agentId={}, kind={}, uid={}, ops={})',
+            '{}(agentId={:.6} kind={} uid={} ops={})',
```

Comment:
Since agentId is 6 (maybe make it 4 to be even more distinguishable) and uid is 8, we can output: 
```txt
JobDriver(wxyz parallel abcdefgh [op()...]
```

We know the line, function, and file for every entry so we have enough context. It's the IDs we need to be able to trace logic.

Reply:
I agree on truncating to 4 characters.

I can do away with ops= since they will say they are ops in the list. Also, eliminating kind= is fine since the value implies the type. 

How strongly do you feel about not having agentId= and uid=? AgentId and uid seem like important info that is not immediately obvious. The person debugging will have the line/function/file for the log message but not the pkdebug_str() function which is where they actually need to go to get the info they are after. 2 hops to find the info (and one hop requiring intricate knowledge of our system) doesn't seem worth it.